src/annotator/integrations/vitalsource.ts

```diff
@@ -22,11 +22,41 @@ import type { InjectConfig } from '../hypothesis-injector';
 // smaller and it feels unreadable or too-zoomed-out
 const MIN_CONTENT_WIDTH = 480;
 
+/**
+ * Book metadata exposed by the VitalSource viewer.
+ */
+type BookInfo = {
+  /**
+   * Indicates the book type. "epub" means the book was created from an EPUB
+   * and the content is XHTML. "pbk" means the book was created from a PDF and
+   * has a fixed layout.
+   */
+  format: 'epub' | 'pbk';
+
+  /**
+   * VitalSource book ID ("vbid"). This identifier is _usually_ the book's
+   * ISBN, hence the field name. However this value is not always a valid ISBN.
+   */
+  isbn: string;
```

Comment:
Given that this is `BookInfo` for a `VitalSource` book specifically, perhaps just calling this `vbid` would make sense.

Reply:
This type is a description of the `<mosaic-book>` API controlled by VitalSource, so it has to use the field names they use.